app/state_machines/verification_attempt_state_machine.rb

```diff
@@ -11,15 +11,15 @@ class VerificationAttemptStateMachine
   transition from: :escalated, to: [:approved, :denied, :requested_replacements]
 
   after_transition(to: :approved, after_commit: true) do |verification_attempt, transition|
-    verification_attempt.client.update(identity_verified_at: transition.created_at)
+    verification_attempt.client.update(identity_verified_at: transition.created_at, identity_verification_denied_at: nil)
```

Comment:
Why do we have to set the identity_verification_denied_at to nil?

Reply:
only one of the options should be filled in at a time, or else you'd be able to be both globally approved and denied which is weird.